src/aegis/src/aegis.erl

```diff
@@ -36,58 +33,40 @@ create(#{} = Db, Options) ->
         db_prefix := DbPrefix
     } = Db,
 
-    % Generate new key
-    DbKey = crypto:strong_rand_bytes(32),
+    {ok, AegisConfig} = aegis_server:generate_key(Db, Options),
 
-    % protect it with root key
-    WrappedKey = aegis_keywrap:key_wrap(?ROOT_KEY, DbKey),
-
-    % And store it
     FDBKey = erlfdb_tuple:pack(?WRAPPED_KEY, DbPrefix),
-    ok = erlfdb:set(Tx, FDBKey, WrappedKey),
+    Packed = erlfdb_tuple:pack({AegisConfig}),
```

Comment:
will this work for arbitrary AegisConfig values or is there an undocument constraint on its form?

Reply:
`erlfdb_tuple:pack/1,2` only accepts tuples (deep tuples) with elements been boolean, int, float or binary. This wrapping here to accommodate possible `false` from key manager.